doc/crypto/api/ops/ka.rst

```diff
@@ -9,11 +9,19 @@
 Key agreement
 =============
 
```

Comment:
Missing changes: in `doc/crypto/appendix/encodings.rst`: “The former can only be used with …”

Reply:
Thank you, good catch.